src/openrct2/world/Map.cpp

```diff
@@ -698,18 +694,18 @@ void map_update_path_wide_flags()
  */
 int32_t map_height_from_slope(int32_t x, int32_t y, int32_t slope)
 {
-    if (!(slope & TILE_ELEMENT_SLOPE_S_CORNER_UP))
+    if (!(slope & FOOTPATH_PROPERTIES_FLAG_IS_SLOPED))
         return 0;
 
-    switch (slope & TILE_ELEMENT_SLOPE_NE_SIDE_UP)
+    switch (slope & FOOTPATH_PROPERTIES_SLOPE_DIRECTION_MASK)
     {
-        case TILE_ELEMENT_SLOPE_FLAT:
+        case 0:
```

Comment:
I understand the change from tile elements to footpath properties, but I don't like the introduction of magic numbers. Could you use or add constants for these?

Reply:
Directions always appear to use naked numbers in OpenRCT2.